pkg/operator/ceph/cluster/mon/node.go

```diff
@@ -22,12 +22,24 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+const (
+	monIPAnnotation = "network.rook.io/mon-ip"
+)
+
 func getNodeInfoFromNode(n v1.Node) (*opcontroller.MonScheduleInfo, error) {
 	nr := &opcontroller.MonScheduleInfo{
 		Name:     n.Name,
 		Hostname: n.Labels[v1.LabelHostname],
 	}
 
+	// If the host networking is setup such that a different IP should be used
+	// than the one that is to the K8s node.dcon
```

Comment:
node.dcon? 
Is this what we are trying to say?

Reply:
hmm, not sure where `dcon` came from, I'll remove it